config/config.go

```diff
@@ -990,6 +1000,12 @@ func SetupViper(v *viper.Viper, filename string, bidderInfos BidderInfos) {
 	v.SetDefault("account_required", false)
 	v.SetDefault("account_defaults.disabled", false)
 	v.SetDefault("account_defaults.debug_allow", true)
+	v.SetDefault("account_defaults.price_floors.enabled", true)
```

Comment:
Should we disable this by default until the feature all parts of the feature are reviewed and merged? Same with `price_floors.enabled`.

Reply:
we can disable